src/Microsoft.OpenApi.Readers/OpenApiStreamReader.cs

```diff
@@ -19,69 +19,47 @@ namespace Microsoft.OpenApi.Readers
     /// </summary>
     public class OpenApiStreamReader : IOpenApiReader<Stream, OpenApiDiagnostic>
     {
-        /// <summary>
-        /// Gets the version of the Open API document.
-        /// </summary>
-        private static string GetVersion(RootNode rootNode)
-        {
-            var versionNode = rootNode.Find(new JsonPointer("/openapi"));
-
-            if (versionNode != null)
-            {
-                return versionNode.GetScalarValue();
-            }
-
-            versionNode = rootNode.Find(new JsonPointer("/swagger"));
-
-            return versionNode?.GetScalarValue();
-        }
+       
 
         /// <summary>
         /// Reads the stream input and parses it into an Open API document.
         /// </summary>
         public OpenApiDocument Read(Stream input, out OpenApiDiagnostic diagnostic)
         {
-            RootNode rootNode;
-            var context = new ParsingContext();
+            ParsingContext context;
```

Comment:
>context [](start = 27, length = 7)

move it into the try?

Reply:
Because the try only catches an exception that is specific to the Yaml parsing library.  So, I actually moved context.Parse out also.